jsonpath4k/build.gradle.kts

```diff
@@ -60,7 +61,7 @@ val generateKotlinGrammarSource = tasks.register<AntlrKotlinTask>("generateKotli
     // The Kotlin target language is implicit, as is the file encoding (UTF-8)
     arguments = listOf("-visitor")
 
-    // Generated files are output inside build/generatedAntlr/{package-name}
+    // Generated files are output inside generatedAntlr/{package-name}
```

Comment:
They are actually put inside the following folder: src/gen/kotlin/{package-name}

Reply:
done